dist/fyta-plant-card.js

```diff
@@ -1082,28 +1109,29 @@ class FytaPlantCard extends LitElement {
         return renderNutrition();
       }
 
+      const sensorSettings = SENSOR_SETTINGS[sensorType];
       const sensorEntityId = this._measurementEntityIds[sensorType];
-      const sensorState = hass.states[sensorEntityId].state;
+      const sensorValue = hass.states[sensorEntityId].state;
+      const formattedSensorValue = this.config.decimals === false ? sensorValue : this._formatDecimals(sensorValue, this.config.decimals);
```

Comment:
```suggestion
      const formattedSensorValue = this.config.decimals === false ? (isNaN(hass.entities[sensorEntityId].display_precision) ? sensorValue : this._formatDecimals(sensorValue, hass.entities[sensorEntityId].display_precision)) : this._formatDecimals(sensorValue, this.config.decimals);
```
I noticed that the display_precision is not used for the custom card. I therefore adjusted the code. Could you please test and confirm it works?